firebase/test/firestore-rules/users.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as rulesTesting from "@firebase/rules-unit-testing";
+import * as utils from "./utils";
+
+describe("users/", () => {
+  const collectionId = "users";
+  let usersRef: utils.CollectionReference;
+  let firestore: utils.ClientFirestore;
+
+  describe("owner users", () => {
+    before(async () => {
+      firestore = await utils.createMyFirestore();
+      usersRef = firestore.collection(collectionId);
+    });
+
+    it("should be able to read its respective document", async () => {
+      await rulesTesting.assertSucceeds(usersRef.doc(utils.myFirestoreUid).get());
+    });
+
+    it("should be denied to read other documents", async () => {
+      await rulesTesting.assertFails(usersRef.get());
+      await rulesTesting.assertFails(usersRef.doc("any").get());
```

Comment:
What's the difference between these asserts?

Reply:
Because there might be rules for the generic `/users` or specific `/users/{value}`